index.bs

```diff
@@ -3089,6 +3095,7 @@ partial interface MLGraphBuilder {
             1. Let |desc1| be a new {{MLOperandDescriptor}}.
             1. Set |desc1|.{{MLOperandDescriptor/dataType}} to |input|'s [=MLOperand/dataType=].
             1. Set |desc1|.{{MLOperandDescriptor/dimensions}} to the [=/list=] « |steps|, |numDirections|, |batchSize|, |hiddenSize| ».
+    1. If running the [=MLActivation/validation steps=] of any [=list/item=] in |options|.{{MLGruOptions/activations}} with |desc0| returns false, then [=exception/throw=] a {{TypeError}}.
```

Comment:
I suppose this step should pass a descriptor with dimensions `[batchSize, hiddenSize]`, because this is the shape of the gates (update, new, reset) where the activations would be applied when running the gruCell().

Reply:
Done in 622ce7535c9a4ded50526a12e905b5bf75b36ecc